### PR TITLE
Edit Remark

### PR DESCRIPTION
**Pull request by scott062**
FE: https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2692
API: https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/839
Mock: https://github.com/MetaPhase-Consulting/mock-fsbid/pull/401

### CODE REVIEW

File: talentmap_api/fsbid/services/common.py [hunk 1]
```diff
@@ -328,7 +328,9 @@ def send_get_request(uri, query, query_mapping_function, jwt_token, mapping_func
 
 def send_put_request(uri, query, query_mapping_function, jwt_token, mapping_function, api_root=API_ROOT):
     mappedQuery = pydash.omit_by(query_mapping_function(query), lambda o: o is None)
+    logger.info(mappedQuery)
```
Comment (pveland621) [inline talentmap_api/fsbid/services/common.py:331]: keeping these in for dev1 testing?

File: talentmap_api/fsbid/services/remark.py [hunk 2]
```diff
@@ -75,23 +74,26 @@ def create_remark_insert(rmrk_seq_num, query, jwt_token):
...
-        'rmrkseqnum': query.get('rmrkseqnum'),
+    formatted_query = {
         'rmrkrccode': query.get('rmrkCategory'),
         'rmrkordernum': 0,
         'rmrkshortdesctext': query.get('shortDescription'),
         'rmrkmutuallyexclusiveind': 'N',
         'rmrktext': query.get('longDescription'),
-        'rmrkactiveind': query.get('rmrkactiveind'),
-        'rmrkcreateid': query.get('rmrkcreateid'),
-        'rmrkcreatedate': query.get('rmrkcreatedate'),
```
Comment (pveland621) [inline talentmap_api/fsbid/services/remark.py:91]: are we getting rid of create date?

File: talentmap_api/fsbid/services/remark.py [hunk 3]
```diff
@@ -108,50 +110,17 @@ def convert_remark_insert_query(query):
...
-
-def edit_remark_insert(rmrk_seq_num, query, jwt_token):
-    '''
-    Create Remark Insert 
-    '''
-    args = {
-        "uri": f"v1/remarks/{rmrk_seq_num}/inserts",
-        "query": query,
-        "query_mapping_function": convert_remark_insert_query,
+        "query_mapping_function": partial(convert_remark_query, is_edit=True),
```
Comment (pveland621) [inline talentmap_api/fsbid/services/remark.py:123]: i forget - when do we use `partial` again?

**Review by pveland621 — APPROVED** [review]
this looks good to me - are we going to go back and handle remark insertions later?

File: talentmap_api/fsbid/services/common.py (hunk 1 above)
Comment (scott062) [reply talentmap_api/fsbid/services/common.py:331]: yep and its a put so we dont do that often

**Comment by scott062** [comment]
> this looks good to me - are we going to go back and handle remark insertions later?

No we are not going to allow for editing of remark inserts

**Pull request merged**